content/root/quick-start-guide.textile

```diff
@@ -41,10 +63,15 @@ ably.connection.on(:connected) do
 end
 
 bc[swift]. let ably = ARTRealtime(key: "{{API_KEY}}")
-ably.connection.on(:connected) do
-  print("That was simple, you're now connected to Ably in realtime")
+ably.connection.on(.Connected) {
+    print("That was simple, you're now connected to Ably in realtime")
```

Comment:
Thanks, I had written that before, so must have lost it in a copy & paste, thanks for picking that up